### ISSUE TITLE
How to pitch ZosPrice?

### ISSUE THREAD

**Issue by SteffenCH** [ZosPrice/issues#1]
I currently dont really have any idea what on earth this is about, besides some pricerunner remake :P 

So maybe first task is to define, what we're actually trying to solve / do?

**Comment by savlo** [comment]
I also do not know shait about this project

**Comment by karbowiak** [comment]
Right, the long short is:

Zos (or whatever it's gonna be called) is a PriceRunner'esque system.
The thought is that everything is stored as plain json files in a GitHub repository - so product data, categories, etc. are all freely available to anyone who want it - or who wants to contribute to it - the secret sauce is just our own indexing and frontend stuff.

This would mean that anyone could use the product data for things, like a M2 module that suggests product information when a new product is made - or someone could make a PIM system that uses said product data - or.. or.. or..

# TLDR:
- Product / Category / Questionnaires / Price Lists / Price History etc. are all stored as .json files that are publicly available
- Frontend is pure JavaScript accessing pre-generated json data (I'm thinking store it all in Cloudflare, and use workers for frontend - and the only live services is Elasticsearch or some shit)
- Backend isn't really a thing, but the backend that exists is only in charge of parsing data from the GitHub repository (.json files) and populating it's own database + search thing (Meilisearch, Elasticsearch, MySQL or whatever)

# Income
Income wise, i'm thinking of giving basically everything away for free - with the only income being
1. Ads on the site
2. Extra functionality, like statistics for a web shop that has their products listed, longer price history, wish lists, "notify me if the price on this product drops" etc. (This could just be a 5-10kr a year price for individuals, and 500-1000kr a year for web shops)

# This GitHub
Idea is for us to basically write down all the ideas we have, in separate issues - talk em through and plan them out - assign tags etc. - put them into the projects view and then execute.

This entire thing is just going to be a side project for those of us interested in it - but one that could be fun to do :)
